botocore/docs/service.py

```diff
@@ -11,11 +11,25 @@
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
 from botocore.docs.bcdoc.restdoc import DocumentStructure
-from botocore.docs.client import ClientDocumenter, ClientExceptionsDocumenter
+from botocore.docs.client import (
+    ClientContextParamsDocumenter,
+    ClientDocumenter,
+    ClientExceptionsDocumenter,
+)
 from botocore.docs.paginator import PaginatorDocumenter
 from botocore.docs.waiter import WaiterDocumenter
 from botocore.exceptions import DataNotFoundError
 
+OMITTED_CONTEXT_PARAMS = {
+    's3': (
+        'Accelerate',
+        'DisableMultiRegionAccessPoints',
+        'ForcePathStyle',
+        'UseArnRegion',
+    ),
+    's3control': ('UseArnRegion',),
+}
+
```

Comment:
Why is this being pulled up to the high level documentation classes instead of staying localized to the `ClientContextParamsDocumenter`?